README.md

```diff
@@ -29,3 +29,39 @@ docker run -p 3000:80 coding-coach
 
 The application, when built and run with docker, runs with `NODE_ENV` set to `production` on http://localhost:3000
 
+## Workflow
+This section describes the workflow we are going to follow when working in a new feature or fixing a bug. If you want to contribute, please follow these steps:
+
+### Fork this project
+Clone the forked project to your local environment, for example:
+`git clone git@github.com:YOUR_GITHUB_USERNAME/coding-coach-api.git` (Make sure to replace the URL to your own repository).
+Add the original project as a remote, for this example the name is upstream, feel free to use whatever name you want.
+`git remote add upstream git@github.com:Coding-Coach/coding-coach-api.git`.
+Forking the project will create a copy of that project in your own GitHub account, you will commit your work against your own repository.
+
+### Update your local copy
+In order to update your local environment to the latest version on development, you will have to pull the changes using the upstream repository, for example: git pull upstream development. This will pull all the new commits from the origin repository to your local environment.
+
+### Features/Bugs/Chores
+When working on a new feature, create a new branch feature/something from the development branch, for example `feature/login-form`. Commit your work agains this new branch and push everything to your forked project. Once everything is completed, you should create a PR to the original project. Make sure to add a description about your work and a link to the trello task.
```

Comment:
There's a small typo in here:
 Commit your work **against**

Reply:
good catch, fixing!